boards/arm/lpcxpresso54114/extra.cmake

```diff
@@ -0,0 +1,11 @@
+#
+# Copyright (c) 2019, NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# This cmake extra file is used for definitions that are custom to this board
+
+
+# Suppress "simple_bus_reg" on LPC boards as all GPIO ports use the same register.
+set(EXTRA_DTC_FLAGS "-Wno-simple_bus_reg")
```

Comment:
list(APPEND

so we don't overwrite any previously set value.

Reply:
@SebastianBoe i see no point to appending, EXTRA_DTC_FLAGS is not used anywhere else, it's there to allow flexibility, but was never set.

Reply:
It could be set by apps for instance.

Reply:
okay will append